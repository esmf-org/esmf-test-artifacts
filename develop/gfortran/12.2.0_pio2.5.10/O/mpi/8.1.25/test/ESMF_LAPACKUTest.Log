2025-05-09 01:57:22
20250509 020834.312 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250509 020834.312 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250509 020834.312 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250509 020834.312 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250509 020834.312 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250509 020834.312 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250509 020834.312 INFO             PET0 Running with ESMF Version   : v8.9.0b08-9-gea72ab2180
20250509 020834.312 INFO             PET0 ESMF library build date/time: "May  9 2025" "01:55:01"
20250509 020834.312 INFO             PET0 ESMF library build location : /glade/derecho/scratch/sacks/esmf-testing/gfortran_12.2.0_pio2.5.10_mpi_O_develop/esmf
20250509 020834.312 INFO             PET0 ESMF_COMM                   : mpi
20250509 020834.312 INFO             PET0 ESMF_MOAB                   : enabled
20250509 020834.312 INFO             PET0 ESMF_LAPACK                 : enabled
20250509 020834.312 INFO             PET0 ESMF_NETCDF                 : enabled
20250509 020834.312 INFO             PET0 ESMF_PNETCDF                : disabled
20250509 020834.312 INFO             PET0 ESMF_PIO                    : enabled
20250509 020834.312 INFO             PET0 ESMF_YAMLCPP                : enabled
20250509 020834.312 INFO             PET0  Beginning Test, file ESMF_LAPACKUTest.F90, line 67
20250509 020834.312 INFO             PET0  NUMBER_OF_PROCESSORS           2
20250509 020834.312 INFO             PET0  PASS DGELSY workspace size inquiry test, ESMF_LAPACKUTest.F90, line 87
20250509 020834.314 INFO             PET0  PASS DGELSY computation test, ESMF_LAPACKUTest.F90, line 98
20250509 020834.314 INFO             PET0  Ending Test, file ESMF_LAPACKUTest.F90, line 109
20250509 020834.314 INFO             PET0  Finalizing ESMF
20250509 020834.311 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250509 020834.311 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250509 020834.311 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250509 020834.311 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250509 020834.311 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250509 020834.311 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250509 020834.311 INFO             PET1 Running with ESMF Version   : v8.9.0b08-9-gea72ab2180
20250509 020834.311 INFO             PET1 ESMF library build date/time: "May  9 2025" "01:55:01"
20250509 020834.311 INFO             PET1 ESMF library build location : /glade/derecho/scratch/sacks/esmf-testing/gfortran_12.2.0_pio2.5.10_mpi_O_develop/esmf
20250509 020834.311 INFO             PET1 ESMF_COMM                   : mpi
20250509 020834.312 INFO             PET1 ESMF_MOAB                   : enabled
20250509 020834.312 INFO             PET1 ESMF_LAPACK                 : enabled
20250509 020834.312 INFO             PET1 ESMF_NETCDF                 : enabled
20250509 020834.312 INFO             PET1 ESMF_PNETCDF                : disabled
20250509 020834.312 INFO             PET1 ESMF_PIO                    : enabled
20250509 020834.312 INFO             PET1 ESMF_YAMLCPP                : enabled
20250509 020834.312 INFO             PET1  Beginning Test, file ESMF_LAPACKUTest.F90, line 67
20250509 020834.312 INFO             PET1  NUMBER_OF_PROCESSORS           2
20250509 020834.312 INFO             PET1  PASS DGELSY workspace size inquiry test, ESMF_LAPACKUTest.F90, line 87
20250509 020834.314 INFO             PET1  PASS DGELSY computation test, ESMF_LAPACKUTest.F90, line 98
20250509 020834.314 INFO             PET1  Ending Test, file ESMF_LAPACKUTest.F90, line 109
20250509 020834.314 INFO             PET1  Finalizing ESMF
