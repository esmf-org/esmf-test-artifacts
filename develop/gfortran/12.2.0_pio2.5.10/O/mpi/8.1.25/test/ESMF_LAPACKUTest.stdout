2025-05-09 01:57:22
  Beginning Test, file ESMF_LAPACKUTest.F90, line 67
  NUMBER_OF_PROCESSORS           2
   suggested workspace length =        4337
  Beginning Test, file ESMF_LAPACKUTest.F90, line 67
  NUMBER_OF_PROCESSORS           2
   suggested workspace length =        4337
  PASS DGELSY workspace size inquiry test, ESMF_LAPACKUTest.F90, line 87
  PASS DGELSY workspace size inquiry test, ESMF_LAPACKUTest.F90, line 87
  PASS DGELSY computation test, ESMF_LAPACKUTest.F90, line 98
  PASS DGELSY computation test, ESMF_LAPACKUTest.F90, line 98
  Ending Test, file ESMF_LAPACKUTest.F90, line 109
  Ending Test, file ESMF_LAPACKUTest.F90, line 109
  PET           0  Test Elapsed Time    6.8679999999999994       msec.
  PET           1  Test Elapsed Time    7.2559999999999985       msec.
